tests/acceptance/editor-selections-test.js

```diff
@@ -57,6 +57,26 @@ test('selecting across sections is possible', (assert) => {
   assert.equal(editor.activeSections.length, 2, 'selects 2 sections');
 });
 
+test('when editing is disabled, the selection detection code is disabled', (assert) => {
+  $('#qunit-fixture').append('<p>outside section</p>');
+
+  editor = new Editor({mobiledoc: mobileDocWithSection});
+  editor.render(editorElement);
+  editor.disableEditing();
+
+  let firstSection = $('p:contains(one trick pony)')[0];
+  let outsideSection = $('p:contains(outside section)')[0];
+
+  Helpers.dom.selectText(editor ,'trick', firstSection,
+                         'outside', outsideSection);
+
+  Helpers.dom.triggerEvent(document, 'mouseup');
+
+  assert.equal(editor.activeSections.length, 0, 'no selection inside the editor');
+  // Edge and IE11 do not introduce newlines
+  assert.equal(Helpers.dom.getSelectedText().replace('\n', ''), "trick ponyoutside");
```

Comment:
This could be `assert.ok(...selectedText.indexOf('trick pony') !== -1 && selectedText.indexOf('outside') !== -1)` to assert that the selection includes inside and outside. Trying to wrangle the selection-to-text differences between browsers is like whack-a-mole. :)